torch/fx/partitioner/nvfuser_operator_support.py

```diff
@@ -0,0 +1,206 @@
+        #     "torch.nn.modules.activation.ReLU": None,
+        #     "torch.nn.modules.activation.Sigmoid": None,
+        #     "torch.nn.modules.activation.Softmax": None,
+        #     "torch.nn.modules.activation.Tanh": None,
+        #     "torch.nn.modules.batchnorm.BatchNorm1d": None,
+        #     "torch.nn.modules.batchnorm.BatchNorm2d": None,
+        #     "torch.nn.modules.conv.Conv2d": None,
+        #     "torch.nn.modules.dropout.Dropout": None,
+        #     "torch.nn.modules.linear.Linear": None,
+        #     "torch.nn.modules.normalization.LayerNorm": None,
+        #     "torch.nn.modules.pooling.AdaptiveAvgPool2d": None,
+        #     "torch.nn.modules.pooling.AvgPool1d": None,
+        #     "torch.nn.modules.pooling.AvgPool2d": None,
+        #     "torch.nn.modules.pooling.AvgPool3d": None,
+        #     "torch.nn.modules.pooling.MaxPool2d": None,
+        #     "torch.nn.modules.rnn.LSTM": None,
+        #     "torch.nn.modules.sparse.Embedding": None,
+        #     "torch.nn.modules.sparse.EmbeddingBag": None,
+        #     # ===============================================================
+        #     # call_method (Tensor)
+        #     # ===============================================================
+        #     "addmm": None,
+        #     "argmax": None,
+        #     "argmin": None,
+        #     "bmm": None,
+        #     "chunk": (([torch.float32],), {}),
+        #     "contiguous": (([torch.float32],), {}),
+        #     "clamp": None,
+        #     "clone": None,
+        #     "dequantize": None,
+        #     "detach": None,
+        #     "eq": (([torch.float32], [torch.float32]), {}),
+        #     "exp": None,
+        #     "flatten": None,
+        #     "float": None,
+        #     "fmod": None,
+        #     "ge": None,
+        #     "gt": None,
+        #     "index_put": None,
+        #     "int": None,
+        #     "le": None,
+        #     "lt": None,
+        #     "log": None,
+        #     "masked_fill": (([torch.float32], [torch.bool]), {}),
+        #     "matmul": None,
+        #     "max": None,
+        #     "min": None,
+        #     "mm": None,
+        #     "ne": (([torch.float32], [torch.float32]), {}),
+        #     "permute": None,
+        #     "pow": None,
+        #     "reshape": None,
+        #     "sigmoid": None,
+        #     "split": None,
+        #     "sqrt": None,
+        #     "sum": None,
+        #     "tanh": None,
+        #     "to": None,
+        #     "topk": None,
+        #     "transpose": (([torch.float32],), {}),
+        #     "type_as": (([torch.bool, torch.float32], [torch.float32]), {}),
+        #     "size": (([torch.float32, torch.bool],), {}),
+        #     "squeeze": None,
+        #     "unsqueeze": (([torch.bool],), {}),
+        #     "view": (([torch.float32],), {}),
+        }
+
+        prim_nvfuser_ops = set(torch._prims.__all__).intersection(dir(fd.Ops))
```

Comment:
No need to take an intersection with `dir(fd.Ops)` here, since the ops should be filtered with `impl_nvfuser is not None`
```suggestion
        prim_ops = torch._prims.__all__
```